src/flag.ts

```diff
@@ -100,6 +101,8 @@ export const monikaFlagsDefaultValue: MonikaFlags = {
   symonGetProbesIntervalMs: 60_000,
   symonReportInterval: DEFAULT_SYMON_REPORT_INTERVAL_MS,
   symonReportLimit: 100,
+  'user-agent':
+    'Monika Synthetic Monitoring Bot Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36',
```

Comment:
I prefer a simpler user-agent with relevant content, such as `monika/{version}`. For instance, `monika/1.20.2`.

Reply:
Changed it to monika/1.20.2, according to the current npm version

Reply:
ooh never mind: recommended is full version:

https://developers.whatismybrowser.com/learn/browser-detection/user-agents/user-agent-best-practices